website/archiver/tasks.py

```diff
@@ -0,0 +1,252 @@
+import requests
+import json
+
+import celery
+from celery.utils.log import get_task_logger
+
+from framework.tasks import app as celery_app
+from framework.tasks.utils import logged
+from framework.exceptions import HTTPError
+
+
+from website.archiver import (
+    ARCHIVER_PENDING,
```

Comment:
Just a thought but you may want to move all statuses into a submodel or dictionary

```python
from website.archiver import statuses

status.checking
status.SUCCESS

# or

status['checking']
status['SUCCESS']
```